modules/ROOT/pages/penetration-testing-policies.adoc

```diff
@@ -9,40 +9,78 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
+. Send your completed Security Assessment form to mailto:securityassessment@salesforce.com[securityassessment@salesforce.com].
++
+Your account representative contacts you when your Security Assessment request is approved.
 
-After you submit your request, you receive a written authorization from the MuleSoft team, at which point you can proceed with testing according to your proposed plan.
+== Run Penetration and Vulnerability Tests  
+
+Because penetration testing could interfere with other tenants, MuleSoft allows penetration testing 
+on your workers but not other Anypoint Platform services.
+
+=== Prerequisites
+
+Before initiating penetration testing, enable static IPs for the app.
+
+By default, CloudHub workers do not use static IP addresses, so you can't test them because their IP addresses might change.
+For information about how to enable static IPs, see xref:deploying-to-cloudhub.adoc#static-ips-tab-settings[Static IPs Tab Settings].
+
+
+=== Steps 
+
+After your Security Assessment request is approved, follow these steps:
+
+. Run the penetration tests.
++
+The Security Assessment Agreement includes restrictions and requirements for testing.
+. Assign a security resource at your company to review and validate findings from the tests.
++
+[IMPORTANT]
+You must complete this step before following up with Salesforce. 
++
+The following documents help the reviewer identify common false positives or security issues related to settings:
++
+--
+** https://help.salesforce.com/articleView?id=Salesforce-Platform-Security-FAQs&type=1&language=en_US[Platform Security FAQ]
+** https://help.salesforce.com[Salesforce Help]
+--
+
+. Send any outstanding security vulnerability findings to mailto:security@salesforce.com[security@salesforce.com].
++
+--
+For information about how to submit a security vulnerability finding, see the 
+https://help.salesforce.com/articleView?id=000320207&type=1&mode=1[Security Vulnerability Finding Submittal Guide].
+
+Include the following information in your email:
+
+** Confirmation number for your Security Assessment approved by Salesforce
+** Summary of all findings and associated severity level of each finding
+** Detailed assessment report noting each finding
+** Steps to reproduce the vulnerability
+** All applicable HTTP requests and responses
+** Explanation of why the example is considered a finding 
+
+[NOTE]
+Salesforce does not address any vulnerabilities found with custom development.
+You must validate and fix any findings with your custom development.
+--
 
 == See Also
 
-* xref:deploying-to-cloudhub.adoc[Deploy to CloudHub]
-* xref:managing-deployed-applications.adoc[Manage Deployed Applications]
+* https://help.salesforce.com/articleView?id=000336416&type=1&mode=1[Salesforce Security vulnerability assessment and penetration test]
```

Comment:
Is this a title; should it be in init. caps.?

Reply:
Nope

Reply:
Sentence case in the target article.